examples/digital_fingerprinting/production/morpheus/benchmarks/README.md

```diff
@@ -16,17 +16,62 @@
 
 # Running DFP E2E Benchmarks
 
-### Set Environment
+### Set up Morpheus Dev Container
 
-To set up and run the benchmarks on production DFP pipeline, follow the instructions provided [here](../../README.md). Once the Morpheus container and the MLflow server have been set up and running with `docker compose`. Attach to the Morpheus pipeline container and download the sample data from S3 per the document's instructions.
+If you don't already have the Morpheus Dev container, run the following to build it:
+```
+./docker/build_container_dev.sh
+```
+
+Now run the container:
+```
+./docker/run_container_dev.sh
+```
+
+Note that Morpheus containers are tagged by date. By default, `run_container_dev.sh` will try to use current date as tag. Therefore, if you are trying to run a container that was not built on the current date, you must set the `DOCKER_IMAGE_TAG` environment variable. For example,
+```
+DOCKER_IMAGE_TAG=dev-221003 ./docker/run_container_dev.sh
+```
+
+In the `/workspace` directory of the container, run the following to compile Morpheus:
+```
+./scripts/compile.sh
+```
+
+Now install Morpheus:
+```
+pip install -e /workspace
+```
+
+Install additonal required dependencies:
 
-## Requirements
-> **Note**: Make sure `gputil`, `dask` and `distributed` are installed in your Conda environment before running the benchmarks. Run the installation command specified below if not.
+```
+mamba install boto3 s3fs
+```
+
+```
+pip install tensorflow
```

Comment:
Instruct the user to install using a conda environment. Try to use an existing one if possible.